.rubocop.yml

```diff
@@ -0,0 +1,80 @@
+require:
+  - rubocop-minitest
+  - rubocop-performance
+
+AllCops:
+  NewCops: disable
+  Exclude:
+    - "bin/**/*"
+    - "Gemfile"
+    - "Gemfile.lock"
+
+Bundler/OrderedGems:
+  Enabled: false
+
+Layout/DotPosition:
+  EnforcedStyle: trailing
+
+Layout/EndOfLine:
+  EnforcedStyle: lf
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Layout/EmptyLinesAroundAccessModifier:
+  EnforcedStyle: only_before
+
+Layout/LineLength:
+  Exclude:
+    - "test/exercises/*"
+    - "lib/analyzers/*/analyze.rb"
+
+Lint/SuppressedException:
+  Exclude:
+    - "test/**/*"
+    - "lib/analyzers/exercise_analyzer.rb"
```

Comment:
This was added to ignore the linting error for [this line](https://github.com/exercism/ruby-analyzer/blob/master/lib/analyzers/exercise_analyzer.rb#L20). Let me know if that is okay

Reply:
Yep, that's fine.

Reply:
That line should probably not be contained within the `begin` and `end` block, as that is provided by the method definition.

```ruby
  def call
    analyze!
    rescue FinishedFlowControlException
    results
  end```